src/routes/postRoutes.ts

```diff
@@ -0,0 +1,7 @@
+import { postController } from '@/controllers/postController';
```

Comment:
Please create a user folder and put this file there to show that it in user routes